recipes/winflexbison/bld.bat

```diff
@@ -0,0 +1,22 @@
+:: MSVC is preferred.
+set CC=cl.exe
+set CXX=cl.exe
+
+mkdir build
+cd build
+cmake ^
+    -G "Ninja" ^
+    -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+    -DCMAKE_BUILD_TYPE=Release ^
+    -DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=True ^
+    -DBUILD_SHARED_LIBS=ON ^
+    %SRC_DIR%
+if errorlevel 1 exit 1
+
+:: Build.
+cmake --build . --config Release
```

Comment:
Can you make this use `%CPU_COUNT%`?

Reply:
`ninja -j%CPU_COUNT%` should work.

Reply:
Thanks. I added the `%CPU_COUNT%`. JFYI, I also found CMake seems to consume a environment variable [`CMAKE_BUILD_PARALLEL_LEVEL`](https://cmake.org/cmake/help/v3.12/envvar/CMAKE_BUILD_PARALLEL_LEVEL.html#envvar:CMAKE_BUILD_PARALLEL_LEVEL) and perhaps `conda-forge-ci-setup` could set it by default in future? What do you think?

Reply:
Thanks for the info. Yes, that would be great to have. PRs are welcome to conda-forge-ci-setup-feedstock.